.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
       - name: Setup Go
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v4
         with:
           go-version: 1.20.x
       - name: Cache build
```

Comment:
Reading the docs for setup-go, in v4 they enable caching by default. This makes me wonder if we need to use `actions/cache` in the following step. I'd remove our cache step and see if it slows things down when you re-run the build (feel free to just push empty commits to test)

Reply:
That's true!
I removed `actions/cache` by 7dd31d40e32789e6aacdd5c9cf2766b820dcdeb8.
I'll see whether it slows things down.

Reply:
I removed `actions/cache` which is used for go's cache.
In this example (lint), removing `actions/cache` eliminates the need to run `Cache build` and `Post Cache build`, resulting in a total reduction of 11 seconds. (The time taken to run `Setup go` remained the same).
So it seems reasonable to remove `actions/cache` which is used for go's cache as you say.

(By the way, it seems to me that it is difficult to compare the total time spent on a job since there are variations as to the total time spent on a job.)

|title|image|job url|
|---|---|---|
|without actions/cache|<img width="1326" alt="image" src="https://github.com/jesseduffield/lazygit/assets/49891479/452833bf-3083-4013-8c36-94994d2bec22">|[job url](https://github.com/jesseduffield/lazygit/actions/runs/5923406537/job/16058979772)|
|with actions/cache|<img width="1344" alt="image" src="https://github.com/jesseduffield/lazygit/assets/49891479/d88b36bb-356f-4174-a2b5-0da3a1bed2af">|[job url](https://github.com/jesseduffield/lazygit/actions/runs/5923559011/job/16059432101)|

Reply:
If there is no problem in above the result, I'll make commit log clean.

Reply:
Looks good!

Reply:
I'm done!